crates/sui-json-rpc-types/src/displays/transaction_displays.rs

```diff
@@ -16,61 +16,75 @@ use tabled::{
 
 impl<'a> Display for Pretty<'a, SuiProgrammableTransactionBlock> {
     fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
-        let mut builder = TableBuilder::default();
-
         let Pretty(ptb) = self;
         let SuiProgrammableTransactionBlock { inputs, commands } = ptb;
-
-        for (i, input) in inputs.iter().enumerate() {
-            match input {
-                SuiCallArg::Pure(v) => {
-                    let pure_arg = if let Some(t) = v.value_type() {
-                        format!(
-                            "{i:<3} Pure Arg: Type: {}, Value: {}",
-                            t,
-                            v.value().to_json_value()
-                        )
-                    } else {
-                        format!("{i:<3} Pure Arg: {}", v.value().to_json_value())
-                    };
-                    builder.push_record(vec![pure_arg]);
-                }
-                SuiCallArg::Object(SuiObjectArg::ImmOrOwnedObject { object_id, .. }) => {
-                    builder.push_record(vec![format!("{i:<3} Imm/Owned Object ID: {}", object_id)]);
-                }
-                SuiCallArg::Object(SuiObjectArg::SharedObject { object_id, .. }) => {
-                    builder.push_record(vec![format!("{i:<3} Shared Object    ID: {}", object_id)]);
-                }
-                SuiCallArg::Object(SuiObjectArg::Receiving { object_id, .. }) => {
-                    builder.push_record(vec![format!("{i:<3} Receiving Object ID: {}", object_id)]);
+        if !inputs.is_empty() {
```

Comment:
Nit: may be nicer to phrase this change in terms of something like 
`if inputs.is_empty() { return write!(EMPTY CASE...}` and then the rest of the code can stay the same. But fine with it as-is as well.

Reply:
right, will do that! Thanks

Reply:
Well, there are two tables being built, so returning early will not allow the second table (commands) to be built.